src/content/quickstart/openshift/_index.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift (AWS)"
+---
+
+## AWS
+
+### openshift-install and pull-secret
+
+Download the **openshift-install** and _oc_ tools, and copy your _pull secret_ from:
+
+> https://cloud.redhat.com/openshift/install/aws/installer-provisioned
```

Comment:
Is it available for people outside RedHat network ?

Reply:
It's for customers, but it should be fine... May be there are even options to try it out.